bears/general/CPDBear.py

```diff
@@ -1,11 +1,9 @@
 from shutil import which
 from xml.etree import ElementTree
-
```

Comment:
don't remove this blank line.As this maintain a gap between absolute import and relative import I guess.